packages/vitest-pool-workers/src/worker/fetch-mock.ts

```diff
@@ -115,12 +118,7 @@ globalThis.fetch = async (input, init) => {
 	});
 
 	// Dispatch the request through the mock agent
-	const dispatchHandlers: Dispatcher.DispatchHandlers = {
-		onConnect(abort) {
-			if (abortSignalAborted) {
-				abort();
-			}
-		},
```

Comment:
This doesn't appear to work in Undici v7. The `abort()` parameter isn't documented so I'm not 100% sure, but in manual testing calling `abort()` here (rather than in `onComplete`) causes `fetch()` calls to abort with `undefined`